recipes/sepp/build.sh

```diff
@@ -1,37 +1,39 @@
 #!/bin/bash
 set -ex
 
-${PYTHON} setup.py config -c
-${PYTHON} setup.py upp -c
-
 # ensure SEPP's configuration file is at the correct location ...
 echo "${PREFIX}/share/sepp/sepp" > home.path
 # ensure directory is created ... ($SP_DIR = Python's site-packages location, see https://docs.conda.io/projects/conda-build/en/stable/user-guide/environment-variables.html#environment-variables-set-during-the-build-process)
-mkdir -p $SP_DIR/
+mkdir -p "${SP_DIR}"
```

Comment:
are double quotes and curly brackets "just" a stylistic change or did you observe errors resulting from *not* having these?